dd-sdk-android-sqldelight/src/test/kotlin/com/datadog/android/sqldelight/DatadogSqliteCallbackTest.kt

```diff
@@ -82,6 +82,7 @@ class DatadogSqliteCallbackTest {
         verify(mockRumMonitor).addError(
             eq(
                 String.format(
+                    Locale.US,
                     DatadogSqliteCallback.DATABASE_CORRUPTION_ERROR_MESSAGE,
                     fakeDbPath,
                     Locale.US
```

Comment:
same, last one should be removed.

Reply:
yeah...bad reg ex replace, good catch, tks :)